konfera/settings.py

```diff
@@ -70,3 +70,14 @@
 How long should we wait to notify about missing payment
 """
 UNPAID_ORDER_NOTIFICATION_REPEAT_DELAY = getattr(settings, 'UNPAID_ORDER_NOTIFICATION_REPEAT_DELAY', 5)
+
+"""
+Absolute url base with protocol
+"""
+ABSOLUTE_URL = getattr(settings, 'ABSOLUTE_URL', 'https://www.pycon.sk/')
```

Comment:
How about we rename it to SITE_URL, you can leave in comment that it is absolute url. I can imagine we will need SITE_NAME for the emails, possibly others as well.